src/lib/election-voters.ts

```diff
@@ -0,0 +1,23 @@
+import { selector } from "atom.io"
+
+import type { ActualVote, SystemUser } from "../types"
+import { currentElectionIdAtom, electionAtom } from "./election"
+import { systemUserAtoms } from "./users"
+import { actualVoteSelectors } from "./votes"
+
+export const currentElectionVotersSelector = selector<{ user: SystemUser; vote: ActualVote }[]>({
+	key: `currentElectionVoters`,
+	get: ({ get }) => {
+		const currentElectionId = get(currentElectionIdAtom)
+		if (currentElectionId == null) {
+			return []
+		}
+		const election = get(electionAtom)
+		const voters = election.users.map((userId) => {
+			const user = get(systemUserAtoms, userId)
+			const vote = get(actualVoteSelectors, userId)
+			return { user, vote }
+		})
+		return voters
```

Comment:
Don't love the name of this variable: `voters` implies an object of the shape `vote[]`, but instead is an array of `{ user, vote }`. Since the name of this selector is `currentElectionVotersSelector`, I would argue that this variable should simply be called `currentElectionVoters`

```suggestion
		const currentElectionVoters = election.users.map((userId) => {
			const user = get(systemUserAtoms, userId)
			const vote = get(actualVoteSelectors, userId)
			return { user, vote }
		})
		return currentElectionVoters
```
though you could also just `return` the `.map` directly rather than assigning it to a variable and then returning it.